pypesto/optimize/optimizer.py

```diff
@@ -555,3 +560,49 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class CmaesOptimizer(Optimizer):
+    """
+    Global optimization using cma-es.
+    """
+
+    def __init__(self, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 10000}
+        self.options = options
+
+    @fix_decorator
+    @time_decorator
+    @history_decorator
+    def minimize(
+            self,
+            problem: Problem,
+            x0: np.ndarray,
+            id: str,
+            history_options: HistoryOptions = None,
+    ) -> OptimizerResult:
+        lb = problem.lb
+        ub = problem.ub
+        sigma0 = 0.25 * np.median(ub)
+        self.options['bounds'] = [lb, ub]
+
+        if cma is None:
+            raise ImportError(
+                "This optimizer requires an installation of cma.")
+
+        result = cma.CMAEvolutionStrategy(
+            x0, sigma0, inopts=self.options
+        ).optimize(problem.objective.get_fval).result
+
+        optimizer_result = OptimizerResult(
+            x=np.array(result[0]),
+            fval=result[1]
+        )
```

Comment:
")" in previous line

Reply:
In the other code segments the indentation of brackets is always like this e.g. pyswarm.
We should then change it everywhere to be consistent.

Reply:
Both is fine for me. A bit of a code-style question ...

Reply:
I fixed it